internal/config/from_env.go

```diff
@@ -90,7 +90,11 @@ func AuthTokenFromEnv(hostname string) (string, string) {
 			return token, GH_ENTERPRISE_TOKEN
 		}
 
-		return os.Getenv(GITHUB_ENTERPRISE_TOKEN), GITHUB_ENTERPRISE_TOKEN
+		if token := os.Getenv(GITHUB_ENTERPRISE_TOKEN); token != "" {
+			return token, GITHUB_ENTERPRISE_TOKEN
+		}
+
+		return os.Getenv(GITHUB_TOKEN), GITHUB_TOKEN
```

Comment:
> for some reason we have an explicit test to ensure that `GITHUB_TOKEN` is not picked up by a non-github.com URL by assuming it's GHES. This PR makes it so that a GHES server would end up reading GITHUB_TOKEN, is there a reason for not doing that?

We intentionally never send GITHUB_TOKEN to non-dotcom instances to avoid ever leaking your `github.com` token anywhere. This should be reverted. https://github.com/github/pse-assessment-reviews/issues/505#issuecomment-686787043

Reply:
@mislav sounds good. The only other solution is that the Codespace will need to send GITHUB_TOKEN under a different name that _would_ be picked up by the CLI (hopefully _that_ is not also a security issue)

Reply:
@mislav this is done